plutus-playground-server/usecases/Crowdfunding.hs

```diff
@@ -107,11 +107,11 @@ instance Scripts.ScriptType Crowdfunding where
     type instance DatumType Crowdfunding = PubKeyHash
 
 scriptInstance :: Campaign -> Scripts.ScriptInstance Crowdfunding
-scriptInstance cmp = Scripts.validator @Crowdfunding
-    ($$(PlutusTx.compile [|| mkValidator ||]) `PlutusTx.applyCode` PlutusTx.liftCode cmp)
+scriptInstance = Scripts.validatorParam @Crowdfunding
+    $$(PlutusTx.compile [|| mkValidator ||])
     $$(PlutusTx.compile [|| wrap ||])
```

Comment:
Ugh, this is still so ugly. If you figure out a way to avoid it I'd be very happy!

Reply:
Yeah, but at least it now looks the same in most examples, with the only difference the type in the type application and the choice between `validator` and `validatorParam`.

Reply:
Yeah, that wasn't a comment on this PR, I just was struck again by its ugliness :(